linode_api4/objects/networking.py

```diff
@@ -67,6 +67,7 @@ class IPAddress(Base):
         "rdns": Property(mutable=True),
         "linode_id": Property(),
         "region": Property(slug_relationship=Region),
+        "vpc_nat_1_1": Property(),
```

Comment:
optional: It might make sense to implement this field as a `JSONObject`

See: https://github.com/linode/linode_api4-python/blob/proj/vpc/linode_api4/objects/linode.py#L299

Reply:
Good point! I named it `InstanceIPNAT1To1` which aligns to the struct naming in linodego that you built.